internal/sqlite_db.go

```diff
@@ -0,0 +1,165 @@
+package internal
+
+import (
+	"database/sql"
+	"errors"
+	"os"
+
+	"github.com/mattn/go-sqlite3"
+)
+
+var (
+	ErrNotExists    = errors.New("row not exist")
+	ErrSetFailed    = errors.New("set failed")
+	ErrDeleteFailed = errors.New("delete failed")
+)
+
+type sqliteDB struct {
+	db *sql.DB
+}
+
+// new instance of sqlite database
+func newSQLiteDB() *sqliteDB {
+	return &sqliteDB{}
+}
+
+// set the connection and filePath of the sqlite db
+func (sqlite *sqliteDB) setConn(filePath string) error {
+	if filePath == "" {
+		return errors.New("no file path provided")
+	}
+
+	os.Remove(filePath)
```

Comment:
this has destructive behavior, losing all data, we shouldn't do that, if we want an option to cleanup previous state it could be parameterized in the function